src/main/java/com/pinterest/secor/timestamp/Kafka8MessageTimestamp.java

```diff
@@ -7,11 +7,11 @@
 
     @Override
     public Long getTimestamp(MessageAndMetadata<byte[], byte[]> kafkaMessage) {
-        return null;
+        return 0l;
     }
 
     @Override
     public Long getTimestamp(MessageAndOffset messageAndOffset) {
-        return null;
+        return 0l;
```

Comment:
Oh yes @HenryCaiHaiying, fixing this.I should have fixed it in previous commit only, my bad. Fixing it.